rfcs/convergence/slot-null-rendering.md

```diff
@@ -0,0 +1,232 @@
+# RFC: Slot null rendering refactoring
+
+---
+
+_@bsunderhus_
+
+## Summary
+
+This RFC proposes a solution for a recurring problem involving slots with no `children` property into 2 steps:
+
+1. `getSlots` should stop verifying null rendering by the existence of the property `children`
+2. Since some slots have default props, verification must be done on shorthand declaration to ensure that those slots will be null rendered (or not) even presenting default props
+
+## Background
+
+`getSlots` is a method that iterates over `ObjectShorthandProps` (_`shorthands`_) declarations and converts them to something that can be rendered by React.
+
+In the case where one _shorthand_ declares a native element (e.g: `"div"`, `"input"`) a verification is made to decide if this declaration will be rendered into something or not (null rendering). This verification will only allow native elements to be rendered if this element presents `children` as property.
```

Comment:
```suggestion
In the case where one _shorthand_ declares a native element (e.g: `"div"`, `"input"`) it will only be rendered if this element has `children` as a property.
```